arangod/Graph/Queues/QueueTracer.cpp

```diff
@@ -147,3 +147,7 @@ template class ::arangodb::graph::QueueTracer<
 /* ClusterServerProvider Section */
 template class ::arangodb::graph::QueueTracer<
     arangodb::graph::FifoQueue<arangodb::graph::ClusterProviderStep>>;
+template class ::arangodb::graph::QueueTracer<
+    arangodb::graph::LifoQueue<arangodb::graph::ClusterProviderStep>>;
+template class ::arangodb::graph::QueueTracer<
+    arangodb::graph::WeightedQueue<arangodb::graph::ClusterProviderStep>>;
```

Comment:
```suggestion
    arangodb::graph::WeightedQueue<arangodb::graph::ClusterProviderStep>>;
    
```

Reply:
We may want to use some generator Makro here as well, but can be done later.

Reply:
Card created: GORDO-1365